components/search/list-renderer/human-donor.js

```diff
@@ -13,10 +13,21 @@ import {
 
 export default function HumanDonor({ item: humanDonor, accessoryData }) {
   const ethnicities =
-    humanDonor.ethnicity?.length > 0 ? humanDonor.ethnicity.join(", ") : "";
+    humanDonor.ethnicities?.length > 0 ? humanDonor.ethnicities.join(", ") : "";
   const sex = humanDonor.sex || "";
   const title = [ethnicities, sex].filter(Boolean);
   const lab = accessoryData?.[humanDonor.lab];
+  const collections =
+    humanDonor.collections?.length > 0
+      ? `${humanDonor.collections.join(", ")}`
+      : "";
+  const phenotypicFeatures = humanDonor.phenotypic_features
+    ?.map((path) => {
+      const feature = accessoryData?.[path];
+      const notes = feature?.notes ? feature?.notes : "Amount";
+      return `${notes} ${feature?.quantity} ${feature?.quantity_units}`;
+    })
+    ?.join(", ");
```

Comment:
If you executed the `.map()` you’re guaranteed to have an array (possibly an empty one), so you don’t really need the question mark before the `.join()`. Makes no execution difference though.

If `feature` doesn’t exist, `notes` could get displayed with two spaces after it. It almost certainly makes no difference, but sometimes it causes future problems if something later gets displayed after this. I think I probably would have used `if (feature)` and then display different things for each case since so many things are conditional on `feature`.